cmd/ui/src/views/Explore/ExploreSearch/CypherInput.tsx

```diff
@@ -37,8 +46,35 @@ const useStyles = makeStyles((theme) => ({
         resize: 'vertical',
         fontSize: '.8rem',
     },
+    cypherEditor: {
+        display: 'flex',
+        flexGrow: 1,
+        flexDirection: 'column',
+        border: '1px solid',
+        borderColor: 'rgba(0,0,0,.23)',
+        borderRadius: theme.shape.borderRadius,
+        backgroundColor: 'white',
+        paddingTop: '5px',
+        minHeight: '120px',
+        // enables drag n drop resizing of editor
+        resize: 'vertical',
+        maxHeight: '500px',
+        overflow: 'auto',
+    },
 }));
 
+const schema = {
+    labels: [
+        ...Object.values(ActiveDirectoryNodeKind).map((nodeLabel) => `:${nodeLabel}`),
+        ...Object.values(AzureNodeKind).map((nodeLabel) => `:${nodeLabel}`),
+    ],
+    relationshipTypes: [
+        ...Object.values(ActiveDirectoryRelationshipKind).map((relationshipType) => `:${relationshipType}`),
+        ...Object.values(AzureRelationshipKind).map((relationshipType) => `:${relationshipType}`),
+    ],
+    propertyKeys: [...Object.values(CommonKindProperties)],
```

Comment:
There are both Active Directory and Azure specific properties that should be included here from `graphSchema.tsx` in the bh-shared-ui src.